Automattic/jetpack
# Comments: ensure form is rendered

Related diff: D126398-code
Related diff: D126723-code

Fixes #

## Proposed changes:

* This PR removes the handling of must login option in our `Highlander_Comments_Base`
* This message should not be visible on Verbum and the form should always show.
* With the second patch Highlander handles this by itself

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Test with simple and atomic sites
* Test with Verbum and Highlander
* The sites you use for testing should have the following discussion options: "Comment author must fill out name and e-mail" and "Users must be registered and logged in to comment" enabled.
* Highlander should show `You must log in to post a comment.`
* Verbum should show the comment form.
